pages/issuer/issue.vue

```diff
@@ -14,17 +14,18 @@
       </ol>
     </nav>
   </div>
-  <div class="flex mt-14">
-    <div class="flex-1 p-4">
+  <div class="flex mt-14 shadow-md min-h-[500px] shadow-md rounded-xl bg-gray-50">
+    <div class="flex-1 p-4 mt-10 ml-10">
       <IssuerDiplomaCredentialForm v-if="step===Step.VC_FORM" @diploma-object-created="createCredentialData" @add-to-log="addToLog" />
-      <div v-if="step!==Step.VC_FORM" class="container mx-auto">
-        <p>Name: {{ CredentialData.signee }}</p>
-        <p>Document Number: {{ CredentialData.documentNumber }}</p>
-        <p>Subject: {{ CredentialData.subject }}</p>
-        <p>Degree: {{ CredentialData.degree }}</p>
-        <p>Date of issue: {{ CredentialData.dateOfIssue }}</p>
-        <p>Message: {{ CredentialData.body }}</p>
-        <br>
+      <div v-if="step!==Step.VC_FORM" class="container mx-auto max-w-4xl bg-white rounded-lg shadow-lg p-6 my-8">
+        <div class="space-y-2">
+          <p class="font-medium text-gray-700">Name: <span class="font-normal">{{ CredentialData.signee }}</span></p>
+          <p class="font-medium text-gray-700">Document Number: <span class="font-normal">{{ CredentialData.documentNumber }}</span></p>
+          <p class="font-medium text-gray-700">Subject: <span class="font-normal">{{ CredentialData.subject }}</span></p>
+          <p class="font-medium text-gray-700">Degree: <span class="font-normal">{{ CredentialData.degree }}</span></p>
+          <p class="font-medium text-gray-700">Date of Issue: <span class="font-normal">{{ CredentialData.dateOfIssue }}</span></p>
+          <p class="font-medium text-gray-700">Message: <span class="font-normal">{{ CredentialData.body }}</span></p>
+        </div>
```

Comment:
Out of curiosity - would that work?

```suggestion
        <div class="space-y-2font-medium text-gray-700">
          <p>Name: <span class="font-normal">{{ CredentialData.signee }}</span></p>
          <p>Document Number: <span class="font-normal">{{ CredentialData.documentNumber }}</span></p>
          <p>Subject: <span class="font-normal">{{ CredentialData.subject }}</span></p>
          <p>Degree: <span class="font-normal">{{ CredentialData.degree }}</span></p>
          <p>Date of Issue: <span class="font-normal">{{ CredentialData.dateOfIssue }}</span></p>
          <p>Message: <span class="font-normal">{{ CredentialData.body }}</span></p>
        </div>
```

Reply:
Yep, it works! I updated it.